[2025-02-05T01:08:41.805+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-05T01:08:41.821+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: northwind_loader.load_postgres manual__2025-02-05T01:08:40.857385+00:00 [queued]>
[2025-02-05T01:08:41.828+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: northwind_loader.load_postgres manual__2025-02-05T01:08:40.857385+00:00 [queued]>
[2025-02-05T01:08:41.828+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-05T01:08:41.840+0000] {taskinstance.py:2889} INFO - Executing <Task(DockerOperator): load_postgres> on 2025-02-05 01:08:40.857385+00:00
[2025-02-05T01:08:41.845+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=10133) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-05T01:08:41.846+0000] {standard_task_runner.py:72} INFO - Started process 10134 to run task
[2025-02-05T01:08:41.846+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'northwind_loader', 'load_postgres', 'manual__2025-02-05T01:08:40.857385+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/northwind_etl/northwind_loader_mel.py', '--cfg-path', '/tmp/tmp4t26172n']
[2025-02-05T01:08:41.847+0000] {standard_task_runner.py:105} INFO - Job 55: Subtask load_postgres
[2025-02-05T01:08:41.880+0000] {task_command.py:467} INFO - Running <TaskInstance: northwind_loader.load_postgres manual__2025-02-05T01:08:40.857385+00:00 [running]> on host 09d05643d017
[2025-02-05T01:08:41.951+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gabriel' AIRFLOW_CTX_DAG_ID='northwind_loader' AIRFLOW_CTX_TASK_ID='load_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-02-05T01:08:40.857385+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-05T01:08:40.857385+00:00'
[2025-02-05T01:08:41.952+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-05T01:08:42.010+0000] {docker.py:379} INFO - Starting docker container from image meu-meltano
[2025-02-05T01:08:42.550+0000] {docker.py:73} INFO - Iniciando processo...
[2025-02-05T01:08:42.553+0000] {docker.py:73} INFO - Processando
[2025-02-05T01:08:42.970+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-05T01:08:42.970+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=northwind_loader, task_id=load_postgres, run_id=manual__2025-02-05T01:08:40.857385+00:00, execution_date=20250205T010840, start_date=20250205T010841, end_date=20250205T010842
[2025-02-05T01:08:43.024+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-05T01:08:43.038+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-05T01:08:43.040+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
